### PR TITLE
Fix sk8 to pass conformance

### PR DESCRIPTION
**Pull request by yastij**
This change updates sk8 to pass conformance, change is minimal and commits are reviewable separately

cc @akutz

### CODE REVIEW

File: e2e/entrypoint.sh [hunk 1]
```diff
@@ -590,7 +590,7 @@ turn_up() {
   if [ "${EXTERNAL}" = "true" ]; then
     printf "waiting for cluster to finish coming online... "
     i=0 && while true; do
-      [ "${i}" -ge 100 ] && { error "timed out waiting for cluster" 1; return; }
+      [ "${i}" -ge 110 ] && { error "timed out waiting for cluster" 1; return; }
```
Comment (akutz) [inline e2e/entrypoint.sh]: Hi @yastij,

Hmmm, a loop iteration count of 100 equals 5 minutes. Should we just double it to 200 for 10 minutes? Any reason not to?

File: sk8.sh [hunk 2]
```diff
@@ -2574,8 +2574,7 @@ EOF
   fi
 
   cat <<EOF >/etc/default/kubelet
-KUBELET_OPTS="--allow-privileged \\
---client-ca-file='${TLS_CA_CRT}'${EXT_CLOUD_PROVIDER_OPTS} \\
+KUBELET_OPTS="--client-ca-file='${TLS_CA_CRT}'${EXT_CLOUD_PROVIDER_OPTS} \\
```
Comment (akutz) [inline sk8.sh]: Hi @yastij,

Any negative affect on a versions of Kubernetes built before [this commit](https://github.com/kubernetes/kubernetes/pull/77820)?

**Comment by akutz** [comment]
Hi @yastij,

One last note -- there are five locations in `sk8.sh` where `-gt 100` is used. Should those be updated as well do you think?

File: sk8.sh (hunk 2 above)
Comment (yastij) [reply sk8.sh]: this should be fine as it already defaults to true to supported version see kubernetes/kubernetes#63442

File: e2e/entrypoint.sh (hunk 1 above)
Comment (yastij) [reply e2e/entrypoint.sh]: I think we'll be fine with this. I can revisit if not enough, but nothing against.

**Comment by yastij** [comment]
@akutz - I think that the others are fine, it's mostly LBs that take time I think

**Pull request merged**